src/docs/Glossary.adoc

```diff
@@ -33,9 +33,15 @@ Scheduler Web Interface:: ProActive component that provides a web interface to t
 [[_glossary_workflow_studio]]
 Workflow Studio:: ProActive component that provides a web interface for designing <<_glossary_workflow,*Workflows*>>.
 
+[[_glossary_catalog]]
+Catalog:: ProActive component that provides storage and versioning of <<_glossary_workflow,*Workflows*>> and other ProActive objects through a REST API. It is also possible to query the catalog for specific <<_glossary_workflow,*Workflows*>>.
```

Comment:
For now it was agreed that we don't explain in documentation about storage of any objects as we don't have GUI for this case. So it will be concentration on workflows for now.
In order to be consistent I will put the same explanation everywhere:
'Catalog stores ProActive Objects and in particular ProActive Workflows'